mailpoet/lib/Automation/Integrations/WooCommerce/Fields/TermOptionsBuilder.php

```diff
@@ -0,0 +1,58 @@
+<?php declare(strict_types = 1);
+
+namespace MailPoet\Automation\Integrations\WooCommerce\Fields;
+
+use MailPoet\Automation\Engine\WordPress;
+use WP_Error;
+use WP_Term;
+
+class TermOptionsBuilder {
```

Comment:
It is in the WooCommerce namespace, but `TermOptionBuilder` still sounds a bit too generic to me. As if I could build term options for all WordPress terms.

Not sure if it makes sense to have a generic `TermOptionBuilder` already/at all and a `WooCommerceTermOptionBuilder` which would use the generic one as dependency :thinking:

Reply:
@websupporter The generic solution is [the private method](https://github.com/mailpoet/mailpoet/pull/4984/files#diff-f83d08b262f774ec342be404b44009815724dada8d48f61d3a2abc86577ae52fR30). Should I make it public and remove the helper methods for categories and tags? Would solve the naming issues as well, I guess.

Reply:
The only thing I would remark is that this could move in a more generic namespace or a `WordPress` namespace. But since we do not have a `WordPress` Integration atm, we can do that later imo.